File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/MinionXpJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import com.google.gson.annotations.Expose;

import java.util.Map;

public class MinionXpJson {
    @Expose
    public Map<String, Xp> minion_xp;

    public static class Xp {
        @Expose
        public String type;

        @Expose
        public Double value;
    }
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/MinionXPJson.kt
package at.hannibal2.skyhanni.data.jsonobjects.repo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class MinionXPJson(
    @Expose @SerializedName("minion_xp") val minionXP: Map<String, Map<String, Double>>,
)
